src/modules/UI/Wallets/action.js

```diff
@@ -84,53 +84,63 @@ export const selectWallet = (walletId: string, currencyCode: string) => (dispatc
   }
 }
 
-function dispatchUpsertWallet (dispatch, wallet, walletId) {
-  dispatch(upsertWallet(wallet))
-  refreshDetails[walletId].delayUpsert = false
-  refreshDetails[walletId].lastUpsert = Date.now()
+function dispatchUpsertWallets (dispatch, wallets: Array<EdgeCurrencyWallet>) {
+  global.pcount('dispatchUpsertWallets')
```

Comment:
@paullinator What is `global.pcount` supposed to do? I am unfamiliar with it.

Reply:
Ack. this snuck in there. It's a debug line that counts the number of calls to that line of code. It's defined in `app.js`. It's safe to leave in, but should be removed.

Reply:
I'll remove it with a future PR since I'm still in this code.